.eslintrc.cjs

```diff
@@ -0,0 +1,9 @@
+module.exports = {
```

Comment:
This file is not needed, since we already have `.eslintrc.js`?

Reply:
Of course. It's good that at least one of us keeps their eyes open.